geoapi/tasks/external_data.py

```diff
@@ -274,7 +275,7 @@ def import_files_recursively_from_path(session, tenant_id: str, userId: int, sys
     contained in specific-file-format metadata (e.g. exif for images) but instead the location is stored in Tapis
     metadata.
 
-    This method is called by refresh_observable_projects() via import_from_agave
+    This method is called by refresh_watch_content_projects() via import_from_agave
```

Comment:
Pedantic comment 14: would `refresh_projects_watch_content` read a little better?

Reply:
I like it. 💯  updated in e6abc452696186e35253c9b111794b02ef0506ad